openapi/openapi.yaml

```diff
@@ -246,4 +245,64 @@ paths:
         '200':
           description: The current User
         'default':
-          description: Error
+          $ref: '#/components/responses/Error'
+
+components:
+  schemas:
+    Error:
+      title: 'Error'
+      description: "A JSON API Error object"
+      type: object
+      externalDocs:
+        description: 'JSON API Error Specification'
+        url: 'https://jsonapi.org/format/#errors'
+      properties:
+        errors:
+          type: array
+          title: 'ErrorItems'
+          items:
+            $ref: '#/components/schemas/ErrorItem'
+          minItems: 1
+      required:
+        - errors
+
+    ErrorItem:
+      title: 'ErrorItem'
+      description: 'An item returned in the Error "errors" value'
+      externalDocs:
+        description: 'JSON API Error Specification'
+        url: 'https://jsonapi.org/format/#errors'
+      type: object
+      properties:
+        code:
+          description: 'Unique identifier for the error'
+          type: string
+          example: 'not_found'
+        detail:
+          type: string
+          description: 'Error detail'
+          example: 'Record not found.'
+        status:
+          type: string
+          description: 'String representation of HTTP status code'
+          example: '404'
+        source:
+          type: object
+          properties:
+            parameter:
+              description: 'A string indicating which URI query parameter caused the error.'
+              type: string
+            pointer:
+              description: 'A JSON Pointer [RFC6901] to the associated entity in the request document '
+              type: string
+      required:
+        - detail
+        - status
+
+  responses:
+    Error:
+      description: 'Error'
+      content:
+        application/vnd.api+json:
```

Comment:
Please use `application/json`, because Galaxy API does not comply with the JSON:API specification, but using parts of it.

Reply:
http://platform-docs.cloud.paas.psi.redhat.com/rest/content.html#content-types-and-type-negotiation  says "Insights Platform APIs SHOULD conform to the JSON API content negotiation specifications."

https://jsonapi.org/format/#content-negotiation points says 'application/vnd.api+json'

That said,  most insights app still specify 'application/json' except for:
https://github.com/RedHatInsights/compliance-backend/blob/master/swagger/v1/openapi.json#L95
https://github.com/RedHatInsights/vulnerability-engine/blob/master/manager.spec.yaml#L17

'application/json' would be my preference, but I would prefer to keep this pr as is, and if we want to change it to do it in one commit. Mostly so I don't have to rebase everything.